src/js/components/account/login.js

```diff
@@ -0,0 +1,89 @@
+import { API } from "../../api";
+import { setMenuState } from "../../utils/menu";
+import {
+  getDiv,
+  getForm,
+  getInput,
+  getLabel,
+  getSubmitButton,
+} from "../../utils/element-factory";
+
+export class Login {
+  constructor() {
+    const $successTemplate = $(".styles .form-styles .w-form-done").clone();
+    const $failTemplate = $(".styles .form-styles .w-form-fail").clone();
+
+    const defaultBaseUrl = "https://muni-portal-backend.openup.org.za";
```

Comment:
why are we hardcoding this again? Can't we use the API which allows for overriding this for dev and staging?